internal/venice-client-common/src/test/java/com/linkedin/venice/schema/merge/AvroCollectionElementComparatorTest.java

```diff
@@ -0,0 +1,129 @@
+package com.linkedin.venice.schema.merge;
+
+import static com.linkedin.venice.schema.merge.AvroCollectionElementComparator.INSTANCE;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotEquals;
+import static org.testng.Assert.assertTrue;
+
+import com.linkedin.avroutil1.compatibility.AvroCompatibilityHelper;
+import com.linkedin.venice.utils.IndexedHashMap;
+import java.util.Collections;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+import org.testng.annotations.Test;
+
+
+public class AvroCollectionElementComparatorTest {
+  @Test
+  public void testCompareWhenSchemaTypeIsNullShouldReturnZero() {
+    assertEquals(INSTANCE.compare(new GenericData(), new GenericData(), Schema.create(Schema.Type.NULL)), 0);
+  }
+
+  @Test
+  public void testCompareWithStringSchema() {
+    assertEquals(INSTANCE.compare("A", "A", Schema.create(Schema.Type.STRING)), 0);
+    assertTrue(INSTANCE.compare("A", "B", Schema.create(Schema.Type.STRING)) < 0);
+    assertTrue(INSTANCE.compare("Z", "A", Schema.create(Schema.Type.STRING)) > 1);
```

Comment:
0 1 and -1 are the outputs.  > 1 is kind of confusing.

Reply:
For `String` types Avro does byte-by-byte comparison
Here is the doc string for the compare function:`Lexicographically compare bytes. If equal, return zero. If greater-than, return a positive value, if less than return a negative value.`

Reply:
Updated to use <, ==, >